src/App.js

```diff
@@ -0,0 +1,160 @@
+import "./App.css";
+import getGreeting from "./util/greeting";
+import queryLocations from "./util/queryLocations";
+
+function App() {
+  const [locations, setLocations] = useState([]);
+
+  useEffect(() => {
+    queryLocations().then((parsedLocations) => {
+      if (parsedLocations != null) {
+        setLocations(parsedLocations);
+      }
+    });
+  }, []);
+
+  const openLocations = locations.filter((location) => location.isOpen);
+  const closedLocations = locations.filter((location) => !location.isOpen);
+
+  const HeaderText = styled(Typography)({
+    color: "white",
+    padding: 0,
+    fontFamily:
+      '"Zilla Slab", "Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif',
+    fontWeight: 800,
+  });
+
+  const LogoText = styled(Typography)({
+    color: "#dd3c18",
+    padding: 0,
+    fontFamily:
+      '"Zilla Slab", "Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif',
+    fontWeight: 800,
+  });
+
+  const FooterText = styled(Typography)({
+    color: "white",
+    marginBottom: 20,
+    fontSize: 16,
+  });
+
+  const NewBadge = styled(Badge)({
+    fontFamily:
+      '"Zilla Slab", "Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif',
+    fontWeight: 800,
+    marginLeft: 20,
+    marginRight: 25,
+    marginBottom: 4,
+    color: "#19b875"
+  });
+
+  return (
+    <div className="App">
+      <div className="Container">
+        <HeaderText variant="h3">{getGreeting()}</HeaderText>
+        <br></br>
+        <HeaderText variant="subtitle1">
+          <NewBadge
+            badgeContent={<HeaderText variant="body2">New</HeaderText>}
+            color="success"
+          ></NewBadge>
+          You can directly browse menus, specials, soups and more at supported
+          locations
+        </HeaderText>
+        <br></br>
+        <br></br>
+        <Grid container spacing={2}>
+          {openLocations.map(
+            (
+              {
+                name,
+                location,
+                isOpen,
+                short_description,
+                todaysSpecials,
+                menu,
+                acceptsOnlineOrders,
+                statusMsg,
+                todaysSoups,
+              },
+              index
+            ) => (
+              <EateryCard
+                name={name}
+                location={location}
+                isOpen={isOpen}
+                description={short_description}
+                menuURL={menu}
+                acceptsOnlineOrders={acceptsOnlineOrders}
+                statusMsg={statusMsg}
+                todaysSpecials={todaysSpecials || []}
+                todaysSoups={todaysSoups || []}
+                key={index}
```

Comment:
nit: setting the key to the index defeats the purpose of the key.  Just use the name since that should be unique, right?